src/coreclr/src/vm/codeman.cpp

```diff
@@ -6175,7 +6175,7 @@ int NativeUnwindInfoLookupTable::LookupUnwindInfoForMethod(DWORD RelativePc,
         if (RelativePc < pNextFunctionEntry->BeginAddress)
         {
             PTR_RUNTIME_FUNCTION pFunctionEntry = pRuntimeFunctionTable + i;
-            if (RelativePc >= pFunctionEntry->BeginAddress)
+            if (RelativePc >= pFunctionEntry->BeginAddress && RelativePc <= pFunctionEntry->EndAddress)
```

Comment:
Is EndAddress actually a valid address? I would expect this to be the RIP after the "ret" instruction i.e. in the non-reachable filler between adjacent functions that's normally getting filled with the "INT 3" instruction (on Intel). Long story short, should this rather be "<code>&lt;</code>" - especially in view of the fact that it _may_ happen in some cases that there's no filler (i.e. the <code>ret</code> instruction has the three lowest-order RIP bits set to 111) in which case the <code>&lt;=</code> sign could cause a spurious match?

Reply:
Yes that should be `<` thanks!